### DISCUSSION TITLE
Disable Projects Public Page

### DISCUSSION THREAD

**Discussion by TheTraille18** [python-gitlab/python-gitlab#2451] [Q&A]
Does anyone know if there is way to disable a projects public page in the settings. I have been looking at the docs and don't see anything.

**Comment by JohnVillalovos** [comment]
Probably need to look at the upstream docs to see if it is possible. If so and PG doesn't support it then would need to add it.

https://docs.gitlab.com/ee/api/api_resources.html

**Reply by TheTraille18** [reply]
Thanks. Was able to solve this by setting the attribute and using the save function.
